ts/webui/src/App.tsx

```diff
@@ -6,8 +6,13 @@ import { isManagerExperimentPage } from './static/function';
 import NavCon from './components/NavCon';
 import MessageInfo from './components/modals/MessageInfo';
 import { SlideNavBtns } from './components/slideNav/SlideNavBtns';
+const echarts = require('echarts/lib/echarts');
+echarts.registerTheme('my_theme', {
```

Comment:
Maybe "nni_theme" or "nniTheme" is a better name?

Reply:
yes, will change it to nni_theme